azure-pipelines.yml

```diff
@@ -1341,6 +1278,169 @@ jobs:
+    inputs:
+      version: $(BuildParameters.dotnetversion)
+      installationPath: $(Agent.TempDirectory)/dotnet
+  - task: DotNetCoreCLI@2
+    displayName: publish DSC for arm64 (OSX)
+    inputs:
+      command: publish
+      publishWebProjects: false
+      projects: src/dsc/dsc.csproj
+      arguments: -c ${{ parameters.BuildConfiguration }} -r osx-arm64 --self-contained false --verbosity detailed
+      zipAfterPublish: false
+      modifyOutputPath: false
+  - task: Bash@3
+    displayName: Download kubectl for OSX arm64
+    inputs:
+      targetType: 'inline'
+      script: |
+        mkdir -p "$(Build.SourcesDirectory)/src/dsc/bin/${{ parameters.BuildConfiguration }}/$(BuildParameters.dotnetversionforbuild)/osx-arm64/publish/kubectl/osx"
+        curl -L https://dl.k8s.io/release/v1.27.3/bin/darwin/arm64/kubectl -o "$(Build.SourcesDirectory)/src/dsc/bin/${{ parameters.BuildConfiguration }}/$(BuildParameters.dotnetversionforbuild)/osx-arm64/publish/kubectl/osx/kubectl"
+
+  - task: Bash@3
+    displayName: code sign for OSX arm64
+    inputs:
+      targetType: 'inline'
+      script: |
+        codesign --force --deep --sign - "$(Build.SourcesDirectory)/src/dsc/bin/${{ parameters.BuildConfiguration }}/$(BuildParameters.dotnetversionforbuild)/osx-arm64/publish/dsc"
+  - task: CopyFiles@2
+    displayName: Collect files for .zip for arm64 (OSX)
+    inputs:
+      SourceFolder: src/dsc/bin/${{ parameters.BuildConfiguration }}/$(BuildParameters.dotnetversionforbuild)/osx-arm64/publish
+      Contents: >-
+        **/*
+
+        src/resources/license.rtf
+
+        src/vscode/ThirdPartyNotices.txt
+
+        !**/*.pdb
+
+        !**/*.xml
+
+        !**/*.nuspec
+
+        !**/cs/*
+
+        !**/de/*
+
+        !**/es/*
+
+        !**/fr/*
+
+        !**/it/*
+
+        !**/ja/*
+
+        !**/ko/*
+
+        !**/pl/*
+
+        !**/pt-BR/*
+
+        !**/ru/*
+
+        !**/tr/*
+
+        !**/zh-Hans/*
+
+        !**/zh-Hant/*
+      TargetFolder: $(Agent.TempDirectory)/zip/osxarm64
+  - task: ArchiveFiles@2
+    displayName: Create .zip file for arm64 (OSX)
+    inputs:
+      rootFolderOrFile: $(Agent.TempDirectory)/zip/osxarm64
+      includeRootFolder: false
+      sevenZipCompression: 5
+      archiveFile: $(Build.ArtifactStagingDirectory)/zip/lpk-osx-arm64.zip
+
+  - task: PublishBuildArtifacts@1
+    displayName: Generate lks.json for .zip files
+    inputs:
+      PathtoPublish: $(Build.ArtifactStagingDirectory)/zipv2
+      ArtifactName: zipv2
+
+  - task: SdtReport@2
+    inputs:
+      GdnExportAllTools: true
+      GdnExportHtmlFile: true
+  - task: PublishSecurityAnalysisLogs@3
+    inputs:
+      ArtifactName: 'CodeAnalysisLogs'
+      ArtifactType: 'Container'
+      AllTools: true
+      ToolLogsNotFoundAction: 'Standard'
+  - task: PostAnalysis@2
+    inputs:
+      GdnBreakAllTools: true
+      GdnBreakFast: true
+- job: Phase_4
+  displayName: "Generate Lks.json for zipv2"
```

Comment:
Is this the right display name?

Reply:
Yep this is the right name, LKS is old acronym which was used in the build systems and vscode extension uses them to download the CLI binaries.